plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -395,6 +395,24 @@ public void testUnregisterTableAccessControl()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testCreateTableInNonExistentSchemaFails()
```

Comment:
We would like to keep the smoke test as simple as possible. Can we move to `BaseIcebergConnectorTest` instead?

Reply:
Yes, there is a specific reason to have this test in `BaseIcebergConnectorSmokeTest`.  I wanted to test against cloud storage. `BaseIcebergConnectorTest` only covers the Local file system.

Reply:
without this fix:
Using Hive metastore -> files are created and Exception is thrown `Schema (.*) not found`
Using Glue metastore -> files are created and Exception is thrown   `<“Database non_existing_schema_mjmeupcfxy not found. (Service: AWSGlue; Status Code: 400; Error Code: EntityNotFoundException; Request ID: 2225b446-d53f-48f5-9fe9-de4fd3e74e54; Proxy: null)“>`

with this fix: `Schema (.*) not found` exception is thrown for all metastore.

`BaseIcebergConnector` does not verify for glue. So IMO `BaseIcebergConnectorSmokeTest` is best suited.